dll/win32/gdiplus/font.c

```diff
@@ -1236,6 +1236,7 @@ static const LANGID mac_langid_table[] =
     MAKELANGID(LANG_GREEK,SUBLANG_DEFAULT),                  /* TT_MAC_LANGID_GREEK */
     MAKELANGID(LANG_ICELANDIC,SUBLANG_DEFAULT),              /* TT_MAC_LANGID_ICELANDIC */
     MAKELANGID(LANG_MALTESE,SUBLANG_DEFAULT),                /* TT_MAC_LANGID_MALTESE */
+    MAKELANGID(LANG_TOK, SUBLANG_DEFAULT),                /* TT_MAC_LANGID_ESPERANTO */
```

Comment:
Technically it's not esperanto, but let's keep it this way maybe, since there's no equivalent in the freetype TT_MAC_LANGID* defines.
```suggestion
    MAKELANGID(LANG_TOK,SUBLANG_DEFAULT),                    /* TT_MAC_LANGID_ESPERANTO */
```

Reply:
Why did you choose to add it here? Shouldn't it go to line 1317?
`TT_MAC_LANGID_ESPERANTO` is already referenced at line 1316: use `Toki Pona` instead?

Reply:
This file, at least, is WINE code, so use `#ifdef __REACTOS__`.